### PR TITLE
added lab1

### PR DESCRIPTION
**Pull request by wertylu**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      home: NissanGTRR35App(),
+    );
+  }
+}
+
+class NissanGTRR35App extends StatefulWidget {
+  const NissanGTRR35App({super.key});
+
+  @override
+  // ignore: library_private_types_in_public_api
```
Comment (DenysDoskochynskyi) [inline lib/main.dart:22]: Nope please update code to prevent lint error ignore:)

File: lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Padding(
+              padding: const EdgeInsets.all(8),
+              child: Text(_displayText, style: const TextStyle(fontSize: 20)),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(8),
+              child: TextField(
+                key: const Key('inputField'), // Use key in widget constructors
```
Comment (DenysDoskochynskyi) [inline lib/main.dart:58]: Remove comment

Comment (DenysDoskochynskyi) [inline lib/main.dart:58]: You didn't need to use key, but ok

File: lib/main.dart [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Nissan GTR R35 quiz'),
+      ),
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
```
Comment (DenysDoskochynskyi) [inline lib/main.dart:50]: You can remove that generic, children always expect List<Widget>

**Review by DenysDoskochynskyi — COMMENTED** [review]
Currently you have some comments, please fix them on next PR and add extra handling
-> If user provide correct data, please also display a some images of this motor)
Also don't forget to remove image if input are wrong

File: lib/main.dart [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  void _updateDisplayText(String value) {
+    if (value.trim().toUpperCase() == 'VR36') {
+      setState(() {
+        _displayText = 'Bro got knowledge balls';
+      });
+    } else {
+      setState(() {
+        _displayText = 'not wirno';
+      });
+    }
```
Comment (BogdanDidukh2003) [inline lib/main.dart:38]:
```suggestion
    setState(() {
	    final isValueVR36 = value.trim().toUpperCase() == 'VR36'
	    _displayText = isValueVR36
		    ? 'Bro got knowledge balls'
		    : 'not wirno';
        });
    }
}
```

**Review by BogdanDidukh2003 — CHANGES_REQUESTED** [review]

**Pull request merged**